cli/src/commands/runTests.js

```diff
@@ -636,6 +741,13 @@ async function runTests(
       if (testGroup.id.match(pattern) != null) {
         return true;
       }
+
+      const depsList = Object.keys(testGroup.deps);
+      if (depsList.length > 0) {
+        return depsList.some(dep => {
+          return testGroup.deps[dep].some(o => `${dep}_${o}`.match(pattern));
+        });
+      }
```

Comment:
It's not exactly clear what this is doing, add a quick comment?

Reply:
Good point! Took me a while to recall but it's pretty important